RecoLocalCalo/HGCalRecProducers/python/HGCalUncalibRecHit_cfi.py

```diff
@@ -4,6 +4,8 @@
 
 fCPerMIP_v9 = cms.vdouble(2.06,3.43,5.15) #120um, 200um, 300um
 
+fCPerMIP_v10 = cms.vdouble(2.06,3.43,5.15) #120um, 200um, 300um
```

Comment:
the values are identical to ```fCPerMIP_v9```.
If this is not a typo, please use v9 values directly (can be even ```fCPerMIP_v10 = fCPerMIP_v9```)